lib/utilities/settings_manager.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:jog_dog/utilities/settings_file_manager.dart';
+
+class Settings {
+
+  String _theme = "Dark";
+  double _tolerance = 0.05;
+
```

Comment:
Wir haben uns letztendlich dafür entschieden, dass wir in den Settings auch den targetSpeed brauchen, da dieser sonst immer auf den Standartwert spring wenn man den Reiter wechselt oder die App neu aufmacht.

Reply:
done